.github/actions/run-interop-ping-test/action.yml

```diff
@@ -0,0 +1,78 @@
+name: 'libp2p ping interop test'
+description: 'Run the libp2p ping interoperability test suite'
+inputs:
+  test-filter:
+    description: "Filter which tests to run out of the created matrix"
+    required: false
+    default: ""
+  extra-versions:
+    description: "Space-separated paths to JSON files describing additional images"
+    required: false
+    default: ""
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        WORK_DIR=$(realpath "${{ github.action_path }}/../../../multidim-interop")
+        echo "WORK_DIR=$WORK_DIR" >> $GITHUB_OUTPUT
+      shell: bash
+      id: find-workdir
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 17
+
+    - name: Expose GitHub Runtime # Needed for docker buildx to cache properly (See https://docs.docker.com/build/cache/backends/gha/#authentication)
+      uses: crazy-max/ghaction-github-runtime@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Install deps
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: npm ci
+      shell: bash
+
+    - name: Build images
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: make
+      shell: bash
+
+    - name: Run the test
+#      timeout-minutes: 30
+      working-directory: ${{ steps.find-workdir.outputs.WORK_DIR }}
+      run: WORKER_COUNT=2 npm run test -- --extra-version=${{ inputs.extra-versions }} --name-filter=${{ inputs.test-filter }}
```

Comment:
Does multiple extra versions work? if I pass in "a.json b.json" does this get interpolated as:
```
 WORKER_COUNT=2 npm run test -- --extra-version=a.json b.json --name-filter=${{ inputs.test-filter }}
 ```
 or 
 ```
  WORKER_COUNT=2 npm run test -- --extra-version="a.json b.json" --name-filter=${{ inputs.test-filter }}
 ```

Reply:
According to the yargs docs yes. I'll do some local testing to confirm.

Reply:
Tested this:
```
❯ npm run test -- --extra-version=a.json b.json   

> Multidimensional libp2p interop test@0.0.1 test
> ts-node testplans.ts --extra-version=a.json b.json

Extra version paths:  [ 'a.json', 'b.json' ]
```

I believe the action will run it without the `"`. so this should work.